src/user/views/adsManager/views/advanced/components/campaign/fields/BudgetField.tsx

```diff
@@ -69,17 +82,38 @@ export function BudgetField() {
           disabled={!isDraft && !advertiser.selfServiceSetPrice}
         />
 
-        {!advertiser.selfServiceSetPrice ? (
-          <Typography variant="body2">
-            {uiTextForCampaignFormat(values.format)} campaigns are priced at a
-            flat rate of{" "}
-            <strong>
-              ${values.price} {_.upperCase(values.billingType)}
-            </strong>
-            .
-          </Typography>
-        ) : (
-          <Stack direction="column" spacing={2} alignItems="flex-start">
+        <Stack direction="column" spacing={2} alignItems="flex-start">
```

Comment:
Is there any way we can simplify this, either by splitting into smaller components or moving some of the logic into (tested) functions?  It's really quite hard to read, and the `prices.find` logic should not be duplicated.